File: app/src/main/java/com/song/deviceinfo/ui/oneline/OneLineViewModel.java
package com.song.deviceinfo.ui.oneline;

import com.song.deviceinfo.ui.base.BaseViewModel;

import java.util.List;

/**
 * Created by chensongsong on 2021/9/10.
 */
public abstract class OneLineViewModel extends BaseViewModel {

    protected abstract List<String> getOneLineInfo();

}

File: app/src/main/java/com/song/deviceinfo/ui/oneline/OneLineViewModel.kt
package com.song.deviceinfo.ui.oneline

import com.song.deviceinfo.ui.base.BaseViewModel

/**
 * Created by chensongsong on 2021/9/10.
 */
abstract class OneLineViewModel : BaseViewModel<String>() {
    abstract val oneLineInfo: List<String>
}
